wfe2/wfe.go

```diff
@@ -2273,6 +2292,45 @@ func (wfe *WebFrontEndImpl) NewOrder(
 
 	logEvent.DNSNames = names
 
+	if features.Get().CheckIdentifiersPaused {
```

Comment:
This `if` stanza seems big enough that it should be factored into its own method. That would also make it easier to write a test for it.